src/lib.rs

```diff
@@ -222,7 +224,10 @@ impl Ffa {
     pub fn msg_wait(&self) -> Result<FfaMsg> {
         let msg = FfaMsg {
             function_id: FfaFunctionId::FfaMsgWait.into(),
-            ..Default::default()
+            source_id: Default::default(),
+            destination_id: Default::default(),
+            uuid: Default::default(),
+            args64: Default::default(),
         };
```

Comment:
what's the point here? Ends up with the same result, no?

Reply:
`FfaMsg` would need to implement `Default`, and for that to be possible, `FfaFunctionId` would need to implement `Default`, but it cannot as there is no reasonable default value for a function ID (we would always want that to be explicit).